openhands/utils/prompt.py

```diff
@@ -205,6 +205,30 @@ def enhance_message(self, message: Message) -> None:
                 micro_text += '\n</extra_info>'
                 message.content.append(TextContent(text=micro_text))
 
+    def enhance_first_user_message(
+        self,
+        message: Message,
+        enable_prompt_extensions: bool,
+    ) -> None:
+        """
+        Add example_message and additional_info into the first user message.
+        """
+        # Get additional info, if enabled
+        additional_info = None
+        if enable_prompt_extensions:
+            additional_info = self.get_additional_info() or None
+
+        # And insert the new content at the start of the list
+        if additional_info:
+            message.content.insert(0, TextContent(text=additional_info))
```

Comment:
Per above, maybe we rewrite `self.get_additional_info()` to `self.add_info_to_first_message()`